nettacker/modules/vuln/crushftp_cve_2025_31161.yaml

```
@@ -0,0 +1,58 @@
+info:
+  name: crushftp_cve_2025_31161_vuln
+  author: OWASP Nettacker Team
+  severity: 9.8
+  description: >  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)